commands/zadd.md

```diff
@@ -10,17 +10,21 @@ members is created, like if the sorted set was empty. If the key exists but does
 
 The score values should be the string representation of a double precision floating point number. `+inf` and `-inf` values are valid values as well.
 
-ZADD options (Redis 3.0.2 or greater)
+ZADD options
 ---
 
 ZADD supports a list of options, specified after the name of the key and before
 the first score argument. Options are:
 
 * **XX**: Only update elements that already exist. Never add elements.
 * **NX**: Don't update already existing elements. Always add new elements.
+* **LT**: Only update existing elements if the new score is **less than** the current score. Adds new elements unless used with **XX**.
+* **GT**: Only update existing elements if the new score is **greater than** the current score. Adds new elements unless used with **XX**.
```

Comment:
I think that mixing XX into that description is harder to understand.
i guess the intention was that it updates only if less/greater than, but can also add new one.
```suggestion
* **LT**: Only update existing elements if the new score is **less than** the current score. This flag doesn't prevent adding new elements if one doesn't exist.
* **GT**: Only update existing elements if the new score is **greater than** the current score. This flag doesn't prevent adding new elements if one doesn't exist.
```

Reply:
You could drop the "if one doesn't exist" part, since that's the definition of a new element.

Reply:
updated.. does that reflect the original intent better? (better than "unless used with XX")

Reply:
I think that's a good phrasing.  It was difficult to find a short way to say that.

As for how the options are arranged, it will probably show up like this:
**ZADD key [NX|XX] [GT|LT] [CH] [INCR] score member [score member ...]**

I figured that was simpler than trying to do the technically more accurate:
**ZADD key [NX|[[XX] [GT|LT]]] [CH] [INCR] score member [score member ...]**

Reply:
I agree